<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE properties SYSTEM "cpf.dtd">
<properties>
<simple name="Import" type="string"><value>maestro</value></simple>

<struct name="ROSHuboCmdOut" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>hubo_cmd</value></simple>
</struct>
<struct name="ROSConStringIn" type="ConnPolicy">
  <simple name="type" type="short"><value>0</value></simple><!-- type of connection: 0 means Data -->
  <simple name="size" type="short"><value>1</value></simple><!-- buffer size -->
  <simple name="transport" type="short"><value>3</value></simple><!--3 means ROS-->
  <simple name="name_id" type="string"><value>ros_in</value></simple>
</struct>


<struct name="RobotControl" type="RobotControl">
  <struct name="Activity" type="PeriodicActivity">
    <simple name="Period" type="double"><value>1</value></simple>
    <simple name="Priority" type="short"><value>0</value></simple>
    <simple name="Scheduler" type="string"><value>ORO_SCHED_RT</value></simple>
  </struct>
  <simple name="AutoConf" type="boolean"><value>1</value></simple>
  <simple name="AutoStart" type="boolean"><value>1</value></simple>
  <struct name="Ports" type="PropertyBag">
    <simple name="hubo_cmd" type="string"><value> ROSHuboCmdOut </value></simple>
    <simple name="ros_in" type="string"><value> ROSConStringIn </value></simple>
  </struct>
  <simple name="Service" type="string"><value>rosparam</value></simple>
</struct>


</properties>
